pages/common/exfatlabel.md

```diff
@@ -0,0 +1,8 @@
+# exfatlabel
+
+> Set an exfat filesystem label.
+> More information: <https://www.mankier.com/8/exfatlabel>.
```

Comment:
```suggestion
> More information: <https://manned.org/exfatlabel>.
```
We prefer using manned when there isn't any documentation from the author. (Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links)